src/connection.toit

```diff
@@ -47,14 +42,15 @@ class Connection:
   close -> none:
     if socket_:
       socket_.close
+      // TODO(florian): should we close the writer?
```

Comment:
That feels like the wrong todo. Maybe rephrase to something like "figure out if we actually want to close the writer here" or "should we avoid closing the writer".